package.json

```diff
@@ -8,36 +8,30 @@
     "url": "https://github.com/Forsakringskassan/eslint-config.git"
   },
   "license": "MIT",
-  "author": "FK IT",
+  "author": "Försäkringskassans",
   "workspaces": [
     "packages/*"
   ],
   "scripts": {
-    "build": "true",
     "eslint": "eslint .",
     "eslint:fix": "eslint --fix .",
     "prettier:check": "prettier --check .",
     "prettier:write": "prettier --write .",
     "test": "npm run eslint && npm run prettier:check"
   },
-  "prettier": "@forsakringskassan/prettier-config",
   "release": {
     "extends": "@forsakringskassan/semantic-release-monorepo-config"
   },
-  "jest": {
-    "preset": "@forsakringskassan/jest-config",
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "example.spec.ts"
-    ]
-  },
   "devDependencies": {
-    "@forsakringskassan/jest-config": "29.3.2",
-    "@forsakringskassan/prettier-config": "1.3.11",
```

Comment:
Temporary removing these until the packages are all published to the public registry (config is inlined in `.prettierrc.cjs`)